emuu-client/src/components/UserAuthentication/newloginscreen.js

```diff
@@ -48,7 +50,7 @@ function Login() {
       .catch((error) => {
         const errorCode = error.code;
         const errorMessage = error.message;
-        console.log("Invalid User Credentials");
+        console.log(errorMessage);
```

Comment:
console.logs are useful for personal testing., but we don't need it on main branch. We can remove them all together before merge to main.